concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -813,7 +813,7 @@ Hale-1973-1798-811	Food_and_Clothing-8	salt	n	07a.08	1274	SALT
 Hale-1973-1798-812	Food_and_Clothing-9	oil	n	07a.09	1484	OIL (ORGANIC SUBSTANCE)
 Hale-1973-1798-813	Food_and_Clothing-10	egg	n	07a.10	744	EGG
 Hale-1973-1798-814	Food_and_Clothing-11	yolk	n	07a.11	1556	YOLK
-Hale-1973-1798-815	Food_and_Clothing-12	white	n	07a.12		
+Hale-1973-1798-815	Food_and_Clothing-12	white	n	07a.12	1335	WHITE
```

Comment:
Is this word really a noun in the original word list? If yes, then I might consider unmapping